<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QgsSpinBox" name="mOriginRowsQgsSpinBox"/>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="originColsLabel">
     <property name="text">
      <string>Cols</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QgsSpinBox" name="mOriginColsQgsSpinBox"/>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>105</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="originRowsLabel">
     <property name="text">
      <string>Rows</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="originBandsLabel">
     <property name="text">
      <string>Bands</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsSpinBox" name="mOriginBandsQgsSpinBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsSpinBox</class>
   <extends>QSpinBox</extends>
   <header>qgsspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
